### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds the Landing Page into the app. 

The routes added are-

- `LandingPage` at ("/") at src/routes/landing-page/LandingPage.jsx

[Live preview](https://deploy-preview-1--notestalk.netlify.app/)

image
image

### CODE REVIEW

File: src/routes/landing-page/LandingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="landing-page-container">
+        <div className="landing-page-content-container">
+          <div className="landing-logo-container">
+            <span className="landing-logo primary">Note</span>
+            <span className="landing-logo secondary">Stalk</span>
+          </div>
+          <p className="xx-large bold">Meet your modern</p>
+          <p className="primary xx-large">Note Taking App</p>
+          <p className="large landing-description">
+            Mange your daily tasks and workflow in a modern way and boost your
```
Comment (TejasShekar) [inline src/routes/landing-page/LandingPage.jsx:17]: Typo -> Manage

**Review by TejasShekar — APPROVED** [review]
Good meaningful class names. Looks good to merge.

File: src/routes/landing-page/LandingPage.jsx (hunk 1 above)
Comment (vishalg8454) [reply src/routes/landing-page/LandingPage.jsx:17]: sure

File: src/routes/landing-page/LandingPage.jsx [hunk 2]
```diff
@@ -1,9 +1,32 @@
 import "./landing-page.css";
-
+import "../../style.css";
+import note_svg from "../../assets/note_svg.svg";
```
Comment (arjun-computer-geek) [inline src/routes/landing-page/LandingPage.jsx:3]: You can use absolute path istead of relative.

**Review by arjun-computer-geek — APPROVED** [review]
Everything  is good. Good code of conduct.

File: src/routes/landing-page/LandingPage.jsx (hunk 2 above)
Comment (vishalg8454) [reply src/routes/landing-page/LandingPage.jsx:3]: ok, will take care in future.

**Review by Developers-codz — APPROVED** [review]
Nothing to suggest, Everything looks perfect.

**Pull request merged**